quiz.css

```diff
@@ -0,0 +1,33 @@
+.card {		border: 1px solid lightgray;
```

Comment:
oh whoops, I guess I assumed the colors were the same. Should probably read more closely next time